src/main/ts/ui/Buttons.ts

```diff
@@ -0,0 +1,118 @@
+import { Editor } from "tinymce";
+
+const register = (editor: Editor): void => {
+  const onAction = () => editor.execCommand('mceEnhancedList');
+
+  editor.ui.registry.addButton('shan-enhanced-lists', {
+    type: 'button',
+    icon: 'smiley',
+    tooltip: 'shan-enhanced-list',
+    onAction: onAction
+  });
+
+  editor.ui.registry.addSplitButton('enhanced-list-unordered-list', {
+    icon: 'unordered-list',
+    tooltip: 'Enhanced unordered list',
+    presets: 'listpreview',
+    columns: 3,
+    fetch: (callback) => {
+      const items: unknown[] = [
+        {
+          type: 'choiceitem',
+          icon: 'list-bull-default',
+          text: 'Disc',
+          value: 'disc'
+        },
+        {
+          type: 'choiceitem',
+          icon: 'list-bull-circle',
+          text: 'Circle',
+          value: 'circle'
+        },
+        {
+          type: 'choiceitem',
+          icon: 'list-bull-square',
+          text: 'Square',
+          value: 'square'
+        },
+      ];
+      callback(items);
+    },
+    onAction: () => {
+      editor.execCommand('InsertOrderedList');
+    },
+    onItemAction: (api, value) => {
+      editor.execCommand('InsertOrderedList', false, { 'list-style-type': value });
+    },
```

Comment:
```suggestion
    onAction: () => {
      editor.execCommand('InsertUnorderedList');
    },
    onItemAction: (api, value) => {
      editor.execCommand('InsertUnorderedList', false, { 'list-style-type': value });
    },
```
Small typo I believe.